Assets/Scripts/Tags/NightLightTag.cs

```diff
@@ -3,4 +3,57 @@
 
 public class NightLightTag : MonoBehaviour
 {
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float _timeOn = 0.6f;
+	[SerializeField] float _fadeTime = 1.0f;
+
+	private Light _light;
+	private float _lightIntensity;
```

Comment:
This could be renamed to something like full intensity. Otherwise it's a bit confusing what's happening in the coroutine.